erpnext/manufacturing/report/job_card_summary/job_card_summary.py

```diff
@@ -180,6 +181,12 @@ def get_columns(filters):
 			{"label": _("To Time"), "fieldname": "to_time", "fieldtype": "Datetime", "width": 120},
 			{
 				"label": _("Time Required (In Mins)"),
+				"fieldname": "time_required",
```

Comment:
This is Expected time Required and not actual.
The actual time Required to complete the job is total_time_in_mins

Reply:
ok, I changed the label name. Now, is this ok ?